server/src/test/java/io/druid/query/lookup/LookupReferencesManagerTest.java

```diff
@@ -90,8 +90,22 @@ public void setUp() throws IOException
     );
     container = new LookupExtractorFactoryContainer("v0", lookupExtractorFactory);
     mapper.registerSubtypes(MapLookupExtractorFactory.class);
+    String temporaryPath = temporaryFolder.newFolder().getAbsolutePath();
     lookupReferencesManager = new LookupReferencesManager(
-        new LookupConfig(temporaryFolder.newFolder().getAbsolutePath(), true),
+        new LookupConfig()
```

Comment:
Turns out this change is net negative on code brevity because of tests, then sorry for suggesting it, now I suggest to roll back, because it doesn't make sense.